sample.display.lvgl.gui fails to build for stm32f429i_disc1

**Describe the bug**
`samples/subsys/display/lvgl/sample.display.lvgl.gui` test fails to build for `stm32f429i_disc1`.

**To Reproduce**

```
scripts/twister -v -N -M -p stm32f429i_disc1 -s samples/subsys/display/lvgl/sample.display.lvgl.gui
```

**Expected behavior**

Test builds successfully.

**Impact**
Unclear

**Logs and console output**
```
Loading Zephyr default modules (Zephyr base).
-- Application: /__w/zephyr/zephyr/samples/subsys/display/lvgl
-- CMake version: 3.20.5
-- Found Python3: /usr/bin/python3.8 (found suitable exact version "3.8.10") found components: Interpreter 
-- Cache files will be written to: /__w/zephyr/zephyr/.cache
-- Zephyr version: 3.3.0-rc1 (/__w/zephyr/zephyr)
-- Found west (found suitable version "0.14.0", minimum required is "0.7.1")
-- Board: stm32f429i_disc1
-- Found host-tools: zephyr 0.15.2 (/opt/toolchains/zephyr-sdk-0.15.2)
-- Found toolchain: zephyr 0.15.2 (/opt/toolchains/zephyr-sdk-0.15.2)
-- Found Dtc: /opt/toolchains/zephyr-sdk-0.15.2/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.6.0", minimum required is "1.4.6") 
-- Found BOARD.dts: /__w/zephyr/zephyr/boards/arm/stm32f429i_disc1/stm32f429i_disc1.dts
-- Generated zephyr.dts: /__w/zephyr/zephyr/twister-out/stm32f429i_disc1/samples/subsys/display/lvgl/sample.display.lvgl.gui/zephyr/zephyr.dts
-- Generated devicetree_generated.h: /__w/zephyr/zephyr/twister-out/stm32f429i_disc1/samples/subsys/display/lvgl/sample.display.lvgl.gui/zephyr/include/generated/devicetree_generated.h
-- Including generated dts.cmake file: /__w/zephyr/zephyr/twister-out/stm32f429i_disc1/samples/subsys/display/lvgl/sample.display.lvgl.gui/zephyr/dts.cmake
Parsing /__w/zephyr/zephyr/Kconfig
Loaded configuration '/__w/zephyr/zephyr/boards/arm/stm32f429i_disc1/stm32f429i_disc1_defconfig'
Merged configuration '/__w/zephyr/zephyr/samples/subsys/display/lvgl/prj.conf'
Configuration saved to '/__w/zephyr/zephyr/twister-out/stm32f429i_disc1/samples/subsys/display/lvgl/sample.display.lvgl.gui/zephyr/.config'
Kconfig header saved to '/__w/zephyr/zephyr/twister-out/stm32f429i_disc1/samples/subsys/display/lvgl/sample.display.lvgl.gui/zephyr/include/generated/autoconf.h'
-- The C compiler identification is GNU 12.1.0
-- The CXX compiler identification is GNU 12.1.0
-- The ASM compiler identification is GNU
-- Found assembler: /opt/toolchains/zephyr-sdk-0.15.2/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    TC_RUNID


-- Build files have been written to: /__w/zephyr/zephyr/twister-out/stm32f429i_disc1/samples/subsys/display/lvgl/sample.display.lvgl.gui
[1/352] Preparing syscall dependency handling

[2/352] Generating include/generated/version.h
-- Zephyr version: 3.3.0-rc1 (/__w/zephyr/zephyr), build: v3.3.0-rc1-97-g432ff20a72e1
[3/352] Generating misc/generated/syscalls_subdirs.trigger
[4/352] Generating misc/generated/syscalls.json, misc/generated/struct_tags.json
[5/352] Generating include/generated/syscall_dispatch.c, include/generated/syscall_list.h
[6/352] Generating include/generated/driver-validation.h
[7/352] Generating include/generated/kobj-types-enum.h, include/generated/otype-to-str.h, include/generated/otype-to-size.h
[8/352] Building C object zephyr/CMakeFiles/offsets.dir/arch/arm/core/offsets/offsets.c.obj
[9/352] Generating include/generated/offsets.h
[10/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/dec.c.obj
[11/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc32_sw.c.obj
[12/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc32c_sw.c.obj
[13/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/hex.c.obj
[14/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc8_sw.c.obj
[15/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc7_sw.c.obj
[16/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc16_sw.c.obj
[17/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/multi_heap.c.obj
[18/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/timeutil.c.obj
[19/352] Building C object zephyr/CMakeFiles/zephyr.dir/misc/generated/configs.c.obj
[20/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/fdtable.c.obj
[21/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/printk.c.obj
[22/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/sem.c.obj
[23/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/thread_entry.c.obj
[24/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/cbprintf.c.obj
[25/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/assert.c.obj
[26/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/heap.c.obj
[27/352] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/common/stm32cube_hal.c.obj
[28/352] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/common/soc_config.c.obj
[29/352] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/logging/log_cache.c.obj
[30/352] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/stm32f4/soc.c.obj
[31/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/heap-validate.c.obj
[32/352] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/tracing/tracing_none.c.obj
[33/352] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/cpu_idle.S.obj
[34/352] Generating linker_zephyr_pre0.cmd
[35/352] Generating linker_zephyr_pre1.cmd
[36/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/cbprintf_packaged.c.obj
[37/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/rb.c.obj
[38/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/bitarray.c.obj
[39/352] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/nmi_on_reset.S.obj
[40/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/cbprintf_complete.c.obj
[41/352] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/mpsc_pbuf.c.obj
[42/352] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/swap_helper.S.obj
[43/352] Building C object CMakeFiles/app.dir/src/main.c.obj
[44/352] Linking C static library app/libapp.a
[45/352] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/isr_wrapper.S.obj
[46/352] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/fault_s.S.obj
[47/352] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/logging/log_mgmt.c.obj
[48/352] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/exc_exit.S.obj
[49/352] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/logging/log_core.c.obj
[50/352] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/logging/log_msg.c.obj
[51/352] Building C object zephyr/arch/common/CMakeFiles/isr_tables.dir/isr_tables.c.obj
[52/352] Building C object zephyr/arch/common/CMakeFiles/arch__common.dir/sw_isr_common.c.obj
[53/352] Linking C static library zephyr/arch/common/libisr_tables.a
[54/352] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/nmi.c.obj
[55/352] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/vector_table.S.obj
[56/352] Linking C static library zephyr/arch/common/libarch__common.a
[57/352] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/irq_manage.c.obj
[58/352] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/logging/backends/log_backend_uart.c.obj
[59/352] Generating ../../../include/generated/libc/minimal/strerror_table.h
[60/352] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/logging/log_output.c.obj
[61/352] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/swap.c.obj
[62/352] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/reset.S.obj
[63/352] Building C object zephyr/CMakeFiles/zephyr_pre0.dir/misc/empty_file.c.obj
[64/352] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/prep_c.c.obj
[65/352] Linking C static library zephyr/libzephyr.a
[66/352] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/thread.c.obj
[67/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/atoi.c.obj
[68/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/font/lv_font_montserrat_24.c.obj
[69/352] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/irq_init.c.obj
[70/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/font/lv_font_montserrat_40.c.obj
[71/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtol.c.obj
[72/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtoul.c.obj
[73/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/bsearch.c.obj
[74/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtoll.c.obj
[75/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtoull.c.obj
[76/352] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/fatal.c.obj
[77/352] Linking C static library zephyr/arch/arch/arm/core/aarch32/libarch__arm__core__aarch32.a
[78/352] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/fpu.c.obj
[79/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strncasecmp.c.obj
[80/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strerror.c.obj
[81/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strstr.c.obj
[82/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/qsort.c.obj
[83/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strspn.c.obj
[84/352] Building C object zephyr/soc/arm/common/cortex_m/CMakeFiles/soc__arm__common__cortex_m.dir/arm_mpu_regions.c.obj
[85/352] Linking C static library zephyr/soc/arm/common/cortex_m/libsoc__arm__common__cortex_m.a
[86/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/math/sqrt.c.obj
[87/352] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/scb.c.obj
[88/352] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/thread_abort.c.obj
[89/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/math/sqrtf.c.obj
[90/352] Building C object zephyr/arch/arch/arm/core/aarch32/mpu/CMakeFiles/arch__arm__core__aarch32__mpu.dir/arm_core_mpu.c.obj
[91/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/string.c.obj
[92/352] Building C object zephyr/drivers/interrupt_controller/CMakeFiles/drivers__interrupt_controller.dir/intc_exti_stm32.c.obj
[93/352] Linking C static library zephyr/drivers/interrupt_controller/libdrivers__interrupt_controller.a
[94/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/abort.c.obj
[95/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/malloc.c.obj
[96/352] Building C object zephyr/arch/arch/arm/core/aarch32/mpu/CMakeFiles/arch__arm__core__aarch32__mpu.dir/arm_mpu.c.obj
[97/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/fprintf.c.obj
[98/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/time/gmtime.c.obj
[99/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/exit.c.obj
[100/352] Linking C static library zephyr/arch/arch/arm/core/aarch32/mpu/libarch__arm__core__aarch32__mpu.a
[101/352] Building C object zephyr/drivers/clock_control/CMakeFiles/drivers__clock_control.dir/clock_stm32f2_f4_f7.c.obj
[102/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/stdout_console.c.obj
[103/352] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/sprintf.c.obj
[104/352] Building C object zephyr/drivers/console/CMakeFiles/drivers__console.dir/uart_console.c.obj
[105/352] Linking C static library zephyr/drivers/console/libdrivers__console.a
[106/352] Linking C static library zephyr/lib/libc/minimal/liblib__libc__minimal.a
[107/352] Building C object zephyr/drivers/clock_control/CMakeFiles/drivers__clock_control.dir/clock_stm32_ll_common.c.obj
[108/352] Linking C static library zephyr/drivers/clock_control/libdrivers__clock_control.a
[109/352] Building C object zephyr/drivers/pinctrl/CMakeFiles/drivers__pinctrl.dir/common.c.obj
[110/352] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32f4xx/soc/system_stm32f4xx.c.obj
[111/352] Building C object zephyr/drivers/memc/CMakeFiles/drivers__memc.dir/memc_stm32.c.obj
[112/352] Building C object zephyr/drivers/memc/CMakeFiles/drivers__memc.dir/memc_stm32_sdram.c.obj
[113/352] Building C object zephyr/drivers/gpio/CMakeFiles/drivers__gpio.dir/gpio_stm32.c.obj
[114/352] Linking C static library zephyr/drivers/memc/libdrivers__memc.a
[115/352] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/fault.c.obj
[116/352] Linking C static library zephyr/drivers/gpio/libdrivers__gpio.a
[117/352] Building C object zephyr/drivers/timer/CMakeFiles/drivers__timer.dir/sys_clock_init.c.obj
[118/352] Linking C static library zephyr/arch/arch/arm/core/aarch32/cortex_m/libarch__arm__core__aarch32__cortex_m.a
[119/352] Building C object zephyr/drivers/pinctrl/CMakeFiles/drivers__pinctrl.dir/pinctrl_stm32.c.obj
[120/352] Building C object zephyr/drivers/display/CMakeFiles/drivers__display.dir/display_stm32_ltdc.c.obj
[121/352] Linking C static library zephyr/drivers/pinctrl/libdrivers__pinctrl.a
[122/352] Building C object zephyr/drivers/reset/CMakeFiles/drivers__reset.dir/reset_stm32.c.obj
[123/352] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32f4xx/drivers/src/stm32f4xx_hal.c.obj
[124/352] Linking C static library zephyr/drivers/reset/libdrivers__reset.a
[125/352] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32f4xx/drivers/src/stm32f4xx_hal_rcc_ex.c.obj
[126/352] Building C object zephyr/drivers/timer/CMakeFiles/drivers__timer.dir/cortex_m_systick.c.obj
[127/352] Linking C static library zephyr/drivers/timer/libdrivers__timer.a
[128/352] Building C object zephyr/drivers/display/CMakeFiles/drivers__display.dir/display_ili9341.c.obj
[129/352] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32f4xx/drivers/src/stm32f4xx_hal_rcc.c.obj
[130/352] Building C object zephyr/drivers/display/CMakeFiles/drivers__display.dir/display_ili9xxx.c.obj
FAILED: zephyr/drivers/display/CMakeFiles/drivers__display.dir/display_ili9xxx.c.obj 
ccache /opt/toolchains/zephyr-sdk-0.15.2/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DCORE_CM4 -DHSE_VALUE=8000000 -DKERNEL -DLV_CONF_INCLUDE_SIMPLE=1 -DLV_CONF_PATH=../zephyr/lv_conf.h -DSTM32F429xx -DUSE_FULL_LL_DRIVER -DUSE_HAL_DRIVER -D__PROGRAM_START -D__ZEPHYR_SUPERVISOR__ -D__ZEPHYR__=1 -I../../../../../../../include -Izephyr/include/generated -I../../../../../../../soc/arm/st_stm32/stm32f4 -I../../../../../../../drivers -I../../../../../../../soc/arm/st_stm32/common -I/__w/zephyr/modules/hal/cmsis/CMSIS/Core/Include -I/__w/zephyr/modules/hal/stm32/stm32cube/stm32f4xx/soc -I/__w/zephyr/modules/hal/stm32/stm32cube/stm32f4xx/drivers/include -I/__w/zephyr/modules/hal/stm32/stm32cube/stm32f4xx/drivers/include/Legacy -I/__w/zephyr/modules/hal/stm32/stm32cube/common_ll/include -I/__w/zephyr/modules/lib/gui/lvgl/zephyr/../src -isystem ../../../../../../../lib/libc/minimal/include -isystem /opt/toolchains/zephyr-sdk-0.15.2/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.1.0/include -isystem /opt/toolchains/zephyr-sdk-0.15.2/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.1.0/include-fixed -fno-strict-aliasing -Os -imacros /__w/zephyr/zephyr/twister-out/stm32f429i_disc1/samples/subsys/display/lvgl/sample.display.lvgl.gui/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee --sysroot=/opt/toolchains/zephyr-sdk-0.15.2/arm-zephyr-eabi/arm-zephyr-eabi -imacros /__w/zephyr/zephyr/include/zephyr/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-pic -fno-pie -Werror -fno-asynchronous-unwind-tables -fno-reorder-functions --param=min-pagesize=0 -fno-defer-pop -fmacro-prefix-map=/__w/zephyr/zephyr/samples/subsys/display/lvgl=CMAKE_SOURCE_DIR -fmacro-prefix-map=/__w/zephyr/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/__w/zephyr=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/drivers/display/CMakeFiles/drivers__display.dir/display_ili9xxx.c.obj -MF zephyr/drivers/display/CMakeFiles/drivers__display.dir/display_ili9xxx.c.obj.d -o zephyr/drivers/display/CMakeFiles/drivers__display.dir/display_ili9xxx.c.obj -c /__w/zephyr/zephyr/drivers/display/display_ili9xxx.c
In file included from ../../../../../../../include/zephyr/toolchain.h:50,
                 from ../../../../../../../include/zephyr/sys/errno_private.h:10,
                 from /__w/zephyr/zephyr/lib/libc/minimal/include/errno.h:32,
                 from ../../../../../../../include/zephyr/drivers/gpio.h:18,
                 from /__w/zephyr/zephyr/drivers/display/display_ili9xxx.h:12,
                 from /__w/zephyr/zephyr/drivers/display/display_ili9xxx.c:9:
/__w/zephyr/zephyr/drivers/display/display_ili9341.h:91:9: error: static assertion failed: "ili9341: Error length display function control (DISCTRL) register"
   91 |         BUILD_ASSERT(DT_PROP_LEN(DT_INST(n, ilitek_ili9341), disctrl) == ILI9341_DISCTRL_LEN,      \
      |         ^~~~~~~~~~~~
/__w/zephyr/zephyr/drivers/display/display_ili9xxx.c:426:9: note: in expansion of macro 'ILI9341_REGS_INIT'
  426 |         ILI##t##_REGS_INIT(n);                                                 \
      |         ^~~
../../../../../../../include/zephyr/sys/util_loops.h:1086:9: note: in expansion of macro 'ILI9XXX_INIT'
 1086 |         F(0, __VA_ARGS__)
      |         ^
../../../../../../../include/zephyr/sys/util_internal.h:105:36: note: in expansion of macro 'Z_UTIL_LISTIFY_1'
  105 | #define UTIL_PRIMITIVE_CAT(a, ...) a##__VA_ARGS__
      |                                    ^
/__w/zephyr/zephyr/drivers/display/display_ili9xxx.c:453:9: note: in expansion of macro 'LISTIFY'
  453 |         LISTIFY(DT_NUM_INST_STATUS_OKAY(ilitek_ili##t), ILI9XXX_INIT, (;), t)
      |         ^~~~~~~
/__w/zephyr/zephyr/drivers/display/display_ili9xxx.c:462:1: note: in expansion of macro 'DT_INST_FOREACH_ILI9XXX_STATUS_OKAY'
  462 | DT_INST_FOREACH_ILI9XXX_STATUS_OKAY(9341);
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[131/352] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32f4xx/drivers/src/stm32f4xx_hal_sdram.c.obj
[132/352] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32f4xx/drivers/src/stm32f4xx_ll_fmc.c.obj
[133/352] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32f4xx/drivers/src/stm32f4xx_ll_utils.c.obj
[134/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/core/lv_theme.c.obj
[135/352] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32f4xx/drivers/src/stm32f4xx_ll_spi.c.obj
[136/352] Building C object zephyr/drivers/serial/CMakeFiles/drivers__serial.dir/uart_stm32.c.obj
[137/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/core/lv_obj_class.c.obj
[138/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/core/lv_group.c.obj
[139/352] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32f4xx/drivers/src/stm32f4xx_hal_ltdc.c.obj
[140/352] Building C object zephyr/drivers/spi/CMakeFiles/drivers__spi.dir/spi_ll_stm32.c.obj
[141/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/core/lv_obj.c.obj
[142/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/core/lv_obj_draw.c.obj
[143/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/core/lv_indev.c.obj
[144/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/core/lv_refr.c.obj
[145/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/core/lv_obj_scroll.c.obj
[146/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/core/lv_obj_style.c.obj
[147/352] Building C object modules/lvgl/CMakeFiles/..__modules__lib__gui__lvgl__zephyr.dir/__w/zephyr/modules/lib/gui/lvgl/src/core/lv_obj_pos.c.obj
ninja: build stopped: subcommand failed.
```

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04
 - Toolchain: Zephyr SDK 0.15.2
 - Commit SHA: https://github.com/zephyrproject-rtos/zephyr/commit/432ff20a72e1bfbc95b40c589b55075d4d82f22e

--- a/boards/arm/stm32f429i_disc1/stm32f429i_disc1.dts
+++ b/boards/arm/stm32f429i_disc1/stm32f429i_disc1.dts
@@ -137,7 +137,7 @@
 		timctrlb = [00 00];
 		pwseqctrl = [64 03 12 81];
 		pumpratioctrl = [20];
-		disctrl = [08 82 27];
+		disctrl = [08 82 27 04];
 		vmctrl1 = [45 15];
 		vmctrl2 = [90];
 		enable3g = [00];

